src/shipchain_common/pagination.py

```diff
@@ -0,0 +1,24 @@
+"""
+Copyright 2019 ShipChain, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from rest_framework_json_api.pagination import PageNumberPagination
```

Comment:
One last thing - can you change this to 
`from rest_framework import PageNumberPagination`?
Technically this is where the base class is coming from.

Reply:
The class from `rest_framework.pagination` does return something like this:
```
{
    "count": 4,
    "next": null,
    "previous": null,
    "data": [...}
}
``` 
Instead of this:
```
{
    "meta": {
        "pagination": {
            "page": 1,
            "pages": 1,
            "count": 4
        }
    },
    "links": {
        "first": "http://localhost:8000/api/v1/shipments/{id}/history/?page=1",
        "last": "http://localhost:8000/api/v1/shipments/{id}/history/?page=1",
        "next": null,
        "prev": null
    },
    "data": [...]
}
```
In fact we are using the one from `rest_framework_json_api`
[https://github.com/ShipChain/transmission/blob/master/conf/base.py#L137](url)

Reply:
Ah, my mistake, I was looking at the latest master of `rest_framework_json_api`, where they have removed the custom PageNumberPagination class. (https://github.com/django-json-api/django-rest-framework-json-api/pull/688/files) Didn't consider that it still exists in the version that we are using.

Reply:
Interesting though! We might need to move to `JsonApiPageNumberPagination` which seems to be the future default class.